CODE_OF_CONDUCT.md

```diff
@@ -1,25 +1,4 @@
-
-This code of conduct applies to all spaces provided by the OpenSource project including in code, documentation, issue trackers, mailing lists, chat channels, wikis, blogs, social media and any other communication channels used by the project.
-
-
-**Our open source communities endeavor to:**
-
-* Be Inclusive: We are committed to being a community where everyone can join and contribute. This means using inclusive and welcoming language.
-* Be Welcoming: We are committed to maintaining a safe space for everyone to be able to contribute.
-* Be Respectful: We are committed to encouraging differing viewpoints, accepting constructive criticism and work collaboratively towards decisions that help the project grow. Disrespectful and unacceptable behavior will not be tolerated.
-* Be Collaborative: We are committed to supporting what is best for our community and users. When we build anything for the benefit of the project, we should document the work we do and communicate to others on how this affects their work.
-
-
-**Our Responsibility. As contributors, members, or bystanders we each individually have the responsibility to behave professionally and respectfully at all times. Disrespectful and unacceptable behaviors include, but are not limited to:**
-
-* The use of violent threats, abusive, discriminatory, or derogatory language;
-* Offensive comments related to gender, gender identity and expression, sexual orientation, disability, mental illness, race, political or religious affiliation;
-* Posting of sexually explicit or violent content;
-* The use of sexualized language and unwelcome sexual attention or advances;
-* Public or private harassment of any kind;
-* Publishing private information, such as physical or electronic address, without permission;
-* Other conduct which could reasonably be considered inappropriate in a professional setting;
-* Advocating for or encouraging any of the above behaviors.
-* Enforcement and Reporting Code of Conduct Issues:
-
-Instances of abusive, harassing, or otherwise unacceptable behavior may be reported. [Contact us](mailto:opensource-codeofconduct@amazon.com). All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances.
\ No newline at end of file
+## Code of Conduct
+This project has adopted the [Amazon Open Source Code of Conduct](https://aws.github.io/code-of-conduct).
```

Comment:
nit - should be blank line after headers

Reply:
This should be in sync with [all other repos](https://github.com/search?q=org%3Aopensearch-project+https%3A%2F%2Faws.github.io%2Fcode-of-conduct-faq&type=code) OpenSearch has.